python/pyspark/pandas/data_type_ops/base.py

```diff
@@ -87,7 +92,6 @@ def __init__(self, dtype: Dtype, spark_type: DataType):
         self.spark_type = spark_type
 
     @property
-    @abstractmethod
```

Comment:
`@abstractmethod` is removed in order to pass mypy checks.
Otherwise, 
```
mypy checks failed:
python/pyspark/pandas/internal.py:1050: error: Cannot instantiate abstract class 'DataTypeOps' with abstract attribute 'pretty_name'
python/pyspark/pandas/internal.py:1441: error: Cannot instantiate abstract class 'DataTypeOps' with abstract attribute 'pretty_name'
```

Reference mypy issue: https://github.com/python/mypy/issues/1843.